client/my-sites/plans/ecommerce-trial/wooexpress-plans/style.scss

```diff
@@ -28,7 +28,53 @@
 		font-family: $sans;
 	}
 
-	.plan-features-2023-grid__content .plan-features-2023-grid__table-item .plan-price {
+	.plan-features-2023-grid__content .plan-features-2023-grid__table-item .plan-price,
+	.plan-features-2023-grid__plan-comparison-grid-container .plan-comparison-grid .plan-price {
 		font-family: $sans;
 	}
+
+	.plan-price .plan-price__currency-symbol {
+		font-size: 1.5rem;
+	}
+
+	.plan-features-2023-grid__plan-comparison-grid-container .plan-comparison-grid .wp-brand-font {
+		display: none;
+	}
+
+	.plan-features-2023-grid__table.has-2-cols {
+		margin-left: auto;
+		margin-right: auto;
+		max-width: 800px;
+	}
+}
+
+.enterprise-ecommerce__banner {
+	align-items: center;
+	border: 1px solid var(--color-border-subtle);
+	border-radius: 4px;
+	display: flex;
+	gap: 1em;
+	justify-content: space-between;
+	margin: 0.5em auto 2em;
+	max-width: calc(800px - 3em);
+	padding: 1em 1.5em 1.5em;
```

Comment:
It seems that the padding distance seems to be slightly different from the design. Perhaps we can directly use pixels (px) or `rem(...px)` so that it's easier to match with Figma.

![Screenshot 2023-06-15 at 12 20 36](https://github.com/Automattic/wp-calypso/assets/4344253/4bab0b76-08fa-49ca-9d4b-6eefd7848f32)
![Screenshot 2023-06-15 at 12 20 55](https://github.com/Automattic/wp-calypso/assets/4344253/41f9cb3f-3e81-496a-b32a-ac2f981b7205)

Reply:
Thanks for catching this; I've updated the styling to match the design